Tutorial/GettingStartedGuide.md

```diff
@@ -0,0 +1,65 @@
+# Getting Started Guide  
+  
+Creating a rest api to expose model's predict functionality  
+  
```

Comment:
Need a few more words. Something along lines of "This guide explains how to package up your trained model as a web service with a predict API that can be scanned using Certifai."